guides/installation_guides/java/windows/README.md

```diff
@@ -1,5 +1,27 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Java Installation Guide
+### Download
+Download the Windows version of Java from the [Java website](https://www.java.com/en/download/) .
+
+### Install
+*Note: administrative access may be required for installation*
+
+1. Run the Java setup file (exe).
+2. Give installation access to the Java installer.
+3. Now give installation path or directory.
+4. Click on Install button.
+
+### Setting up Environmental Variables
+
+1. Right click Computer.
+2. Click Properties.
+3. On the left pane select Advanced System Settings.
+4. Select Environment Variables.
+5. Under the System Variables, Select PATH and click edit.
+6. Then click New.
+7. Then add path as "C:\ProgramFiles\Java\jdk1.8.0_131\bin" (depending on your installation path).
```

Comment:
Put the path in backticks (``). Also change depending on your installation path to depending on your installation path _and_ version.